explain images in cc are same as md/mdx images (#6799)
withastro/docs: src/content/docs/en/guides/images.mdx

> ## Images in Content Collections

Reviewer: ```suggestion
## Images in content collections
```

> Images in Content Collections will be processed the same way they are in [Markdown](#images-in-markdown-files) and [MDX](#images-in-mdx-files) depending on which filetype you are using.

Reviewer 2: ```suggestion
Images in Content Collections will be processed the same way they are in [Markdown](#images-in-markdown-files) and [MDX](#images-in-mdx-files) depending on which file type you are using.
```

Reviewer 2: Since the docs tend to use file type

> Images in Content Collections will be processed the same way they are in [Markdown](#images-in-markdown-files) and [MDX](#images-in-mdx-files) depending on which file type you are using.

Reviewer: ```suggestion
Images in content collections will be processed the same way they are in [Markdown](#images-in-markdown-files) and [MDX](#images-in-mdx-files) depending on which file type you are using.
```

> Additionally you can declare an associated image for a content collections entry, such as a blog post's cover image, in your frontmatter using its path relative to the current folder:

Reviewer: ```suggestion
Additionally, you can declare an associated image for a content collections entry, such as a blog post's cover image, in your frontmatter using its path relative to the current folder:
```
